editor/js/ui/editor.js

```diff
@@ -466,17 +470,23 @@ RED.editor = (function() {
      * @param property - the name of the field
      * @param prefix - the prefix to use in the input element ids (node-input|node-config-input)
      */
-    function preparePropertyEditor(node,property,prefix) {
+    function preparePropertyEditor(node,property,prefix,definition) {
         var input = $("#"+prefix+"-"+property);
         if (input.attr('type') === "checkbox") {
             input.prop('checked',node[property]);
-        } else {
+        } 
+        else {
             var val = node[property];
             if (val == null) {
                 val = "";
             }
-            input.val(val).attr("dir", RED.bidi.resolveBaseTextDir(val));
-            RED.bidi.initInputEvents(input);            
+            if ("format" in definition[property] && definition[property].format !== "" && input[0].nodeType === 1) {
```

Comment:
Both `<input>` and `<div>` have a `nodeType === 1` - so not sure how this is meant to be distinguishing between them.

 I would have expected a test against `nodeName` which would be equal to `INPUT` or `DIV`.